internal/host/transition.go

```diff
@@ -448,13 +448,31 @@ func (th *transitionHandler) HasInstallationTimedOut(sw stateswitch.StateSwitch,
 	if !ok {
 		return false, errors.New("HasInstallationTimedOut incompatible type of StateSwitch")
 	}
+	params, ok := args.(*TransitionArgsRefreshHost)
+	if !ok {
+		return false, errors.New("HasInstallationTimedOut invalid argument")
+	}
+	if isPaused, err := IsInstallationPendingUserAction(sHost.host.ClusterID, params.db); err != nil {
```

Comment:
Not relevant for this stage.
It is used in a transition that validate start of the installation, that we started to run the installer on the host, there is no wrong boot order in this case.